authentication/logout/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="https://main--eager-nightingale-ec3f88.netlify.app/components/component.css">
+    <link rel="stylesheet" href="../../index.css" />
+    <link rel="stylesheet" href="../../common.css" />
+    <link rel="stylesheet" href="./logout.css" />
+</head>
+
+<body>
+    <div class="container">
+        <nav class="header">
```

Comment:
It's good to wrap the main nav in a "header" tag.

Reply:
resolved